findHelpScreens/QuickTipsScreens/QuickTips.js

```diff
@@ -79,12 +79,14 @@ const styles = StyleSheet.create({
       color: colors.primary.white, 
```

Comment:
Looks like every card we have in Find Help is like this:
![image](https://user-images.githubusercontent.com/22266870/79055549-87a23980-7c13-11ea-99af-5550ac4832ad.png)